plugins/org.jkiss.dbeaver.ext.gbase8s/src/org/jkiss/dbeaver/ext/gbase8s/GBase8sUtils.java

```diff
@@ -78,11 +77,12 @@ public static boolean isOracleSqlMode(DBPDataSourceContainer container) {
         }
         DBPDriver driver = container.getDriver();
         Map<String, Object> driverProps = driver.getConnectionProperties();
-        return driverProps.values().stream()
-                .filter(Objects::nonNull)
-                .map(Object::toString)
-                .map(String::trim)
-                .anyMatch(value -> "oracle".equalsIgnoreCase(value));
+        for (String key : driverProps.keySet()) {
+            if ("sqlmode".equalsIgnoreCase(key)) {
+                return "oracle".equalsIgnoreCase(Objects.toString(driverProps.get(key), "").trim());
```

Comment:
The code above doesn't check for `null` values. Maybe it's better to extract that part to avoid repetition?